# Generated by Django 3.1.2 on 2022-12-10 15:04

from django.conf import settings
import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('ElderlyApp', '0003_auto_20221210_1543'),
    ]

    operations = [
        migrations.AddField(
            model_name='usercontact',
            name='emergency_contact',
            field=models.BooleanField(default=False),
        ),
        migrations.CreateModel(
            name='UserData',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('age', models.IntegerField(default=50, validators=[django.core.validators.MaxValueValidator(110)])),
                ('telephone', models.CharField(max_length=10)),
                ('civilian_id', models.CharField(max_length=10)),
                ('home_address', models.CharField(max_length=30)),
                ('surname', models.CharField(max_length=30)),
                ('lastname', models.CharField(max_length=30)),
                ('informations', models.TextField()),
                ('born', models.DateField()),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='user_data', to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'unique_together': {('civilian_id',)},
            },
        ),
    ]
